src/Coverage/CoverageMapProvider.ts

```diff
@@ -41,7 +36,7 @@ export class CoverageMapProvider {
   setFileCoverage(filePath: string, map: CoverageMap): void {
     this._map.data[filePath] = map.fileCoverageFor(filePath);
   }
-  public getFileCoverage(filePath: string): FileCoverage {
+  public getFileCoverage(filePath: string): any {
```

Comment:
TypeScript was complaining about FileCoverage not having a bunch of properties. I couldn't figure out how to add the missing properties, so I fudged it.

Reply:
you can add a type check, something like this:
```
public getFileCoverage(filePath: string): FileCoverage {
    const coverage = this._map.data[filePath];
    if(!(coverage instanceof FileCoverage)){
      throw new Error(`expect a FileCoverage but got ${JSON.stringify(coverage)}`);
    }
    return coverage;
  }
```
we actually only save FileCoverage in the _map, but adding a type-check is better foolproof than just casing.